src/core/scene/a-scene.js

```diff
@@ -323,15 +360,15 @@ module.exports = registerElement('a-scene', {
   })
 });
 
-function getCanvasSize (canvas) {
-  if (isMobile) {
+function getCanvasSize (canvas, embedded) {
```

Comment:
this could use a docstring now on the effects of passing `embedded`, return cases, and why